### DISCUSSION TITLE
Has anyone ever tried Blue Charm Beacons?

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#15] [General]
Like these: https://www.amazon.com/s?k=Blue+Charm+Beacons&me=A3LUSRMOBZQLMS&linkCode=sl2&tag=espresense-20&linkId=e5f23a715a0c66036cd04e0f7deb7050&language=en_US&ref_=as_li_ss_tl

**Comment by Scott8586** [comment]
Yes, I. use them, I can see that espresense is picking up one now (I track my car with it):

```
espresense/rooms/familyroom {"id":"iBeacon:746e6f636165426d7261684365756c42-3838-4949","name":"RAV4","rssi@1m":-64,"rssi":-87,"mac":"806fb06c8ae8","raw":4.54,"distance":4.96,"speed":0}
```

**Comment by DTTerastar** [comment]
Cool, I've got one coming Sat from Amazon.

**Comment by DTTerastar** [comment]
ESPresense can now read the voltage and temp from them.

**Reply by Franz37** [reply]
I see in MQTT explorer that I can read my voltage of my Bluecharm but how can you integrate this value in Home assitant.
In the same way as I can see the attribute "distance" but I did not do anything to get the attribute in Home assistant.

It could be usefull because today I have to had another BLE tracker to get the battery level.

Thanks for the jobb.

**Reply by DTTerastar** [reply]
You add a mqtt sensor and a value template

**Comment by Franz37** [comment]
Thanks for your reply.
I have succeeded today after fighting a bit.

So now I get my battery checked from all the esp throughout the house!
I can skip using the extra  BLE tracker platform from esphome!

Good job guys.
